modules/tribes/client/components/CirclePage.component.js

```diff
@@ -0,0 +1,190 @@
+          <div className="row no-gutters">
+            <div className="col-xs-12">
+              <a
+                onClick={goBack}
+                className="btn btn-lg btn-link tribe-header-back"
+              >
+                <i className="icon-left"></i>
+                {t('More circles')}
+              </a>
+            </div>
+          </div>
+          <div className="row">
+            {/* For authenticated members */}
+            {user && (
+              <div
+                className="col-xs-10 col-sm-offset-1 col-sm-7 col-md-6 col-lg-5"
+                // ng-if="app.user"
+              >
+                <p className="lead tribe-pre">{t('Circle')}</p>
+                <h2 className="font-brand-regular tribe-title">
+                  {circle.label}
+                </h2>
+                <div className="tribe-meta">
+                  {circle.count === 0 && t('No members yet')}
+                  {circle.count === 1 && t('One memeber')}
+                  {circle.count > 1 && t(`${circle.count} members`)}
+                </div>
+                {circle.description && (
+                  <div className="lead tribe-meta">{t(circle.description)}</div>
+                )}
+                <br />
+                <br />
+                <JoinButton tribe={circle} user={user} onUpdated={() => {}} />
+                &nbsp;
+                <a
+                  className="btn btn-lg btn-default"
+                  href={`/search?tribe=${circle.slug}`}
+                  type="button"
+                >
+                  {t('Find members')}
+                </a>
+                <br />
+                <br />
+                <p className="lead tribe-intro">
+                  {t(
+                    'Trustroots is built on communities. Share this page within your community and invite them to join!',
+                  )}
+                </p>
+                {circle.label && (
+                  <>
+                    {/* Twitter share button */}
+                    <div
+                      className="tribe-share"
+                      tr-share-twitter
+                      data-text={t(
+                        `Join to meet, host and get hosted by Trustroots circle ${circle.label} — I'm in!`,
+                      )}
+                    ></div>
+                    {/* FB share button */}
+                    <div className="tribe-share" tr-share-fb></div>
+                  </>
+                )}
+              </div>
+            )}
+            {/* For non authenticated members */}
+            {!user && (
+              <div className="col-xs-12 col-sm-offset-1 col-sm-7 col-md-6 col-lg-5">
+                <p className="lead tribe-pre">{t('Trustroots circle')}</p>
+                <h2 className="font-brand-regular tribe-title">
+                  {t(circle.label)}
+                </h2>
+                <div className="lead tribe-meta">
+                  {circle.count === 0 && t('No members yet')}
```

Comment:
We can use [plurals of i18n-next](https://www.i18next.com/translation-function/plurals) like so:
```js
t('{{count}} members', {count: circle.count})
```